src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -433,34 +433,38 @@ class C
 {
     public void M()
     {
-        void Local([A, B]int x, [CLSCompliant]string s = """") { }
+        void Local([A, B, D(2)]int x, [CLSCompliant]string s = """") { }
```

Comment:
>void Local([A, B, D(2)]int x, [CLSCompliant]string s = """") { } [](start = 8, length = 64)

Please do not bundle tests for new scenarios into existing tests, create separate targeted tests.